package.json

```diff
@@ -14,8 +14,7 @@
     "url": "git+https://github.com/playcanvas/pcui.git"
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js && MODULE=true webpack --config webpack.config.js && MODULE=true MODULE_STRIP=true webpack --config webpack.config.js",
-    "watch": "ENVIRONMENT=development webpack --config webpack.config.js --watch",
+    "build": "rm -r dist; rollup -c; rm dist/index.mjs; rm dist/bundle/index.mjs",
```

Comment:
This will be linux centric again, for Windows its `rimraf`

Reply:
Good point changed that to use rollup instead.